database.py

```diff
@@ -93,6 +93,14 @@ def get_extra(self):
         # Get all the Extra Pokemon images available.
         return self.__get_region(None)
 
+    def get_light(self):
```

Comment:
I agree on the need for extra logic around `escape_code`.  I look forward to seeing light and dark themes.  Great idea.